src/commons/api/history.rs

```diff
@@ -566,9 +581,18 @@ impl WithStorableDetails for StorableCaCommand {
             StorableCaCommand::KeyRollFinish { resource_class_name } => {
                 CommandSummary::new("cmd-ca-keyroll-finish", &self).with_rcn(resource_class_name)
             }
+
+            // ROA
             StorableCaCommand::RoaDefinitionUpdates { updates } => CommandSummary::new("cmd-ca-roas-updated", &self)
                 .with_added(updates.added().len())
                 .with_removed(updates.removed().len()),
+
+            // ASPA
+            StorableCaCommand::AspasUpdate { .. } => CommandSummary::new("cmd-ca-aspas-update", &self),
+            StorableCaCommand::AspasUpdateExisting { .. } => CommandSummary::new("cmd-ca-aspas-update-existing", &self),
+            StorableCaCommand::AspaRemove { .. } => CommandSummary::new("cmd-ca-aspa-remove", &self),
```

Comment:
The other two commands above use "aspas-" but this one uses "aspa-". Should they be the same for consistency?

Reply:
done